I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 138, in script
    python:
  File "game/script.rpy", line 146, in <module>
    response = requests.post('https://tlpscoreboard-2638f0dbf094.herokuapp.com/submit_score/', data=data)
NameError: name 'requests' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 138, in script
    python:
  File "C:\Users\JOHN\Downloads\renpy-8.1.0-sdk\renpy\ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\JOHN\Downloads\renpy-8.1.0-sdk\renpy\python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 146, in <module>
    response = requests.post('https://tlpscoreboard-2638f0dbf094.herokuapp.com/submit_score/', data=data)
NameError: name 'requests' is not defined

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.0.23051307
striped_template 1.0
Mon Jun 19 21:37:40 2023
